manim/mobject/svg/text_mobject.py

```diff
@@ -63,14 +66,19 @@ def __init__(self, start, end, font, slant, weight, line_num=-1):
 
 
 class Text(SVGMobject):
-    """ Text is used for displaying Text.
-    Text is VGroup() of each characters
-    that mean you can use it like 
-        Text[0:5] or Text.chars[0:5] to access first five characters 
-    Text[0:5] or Text or Text.chars[0:5] will create problems when using Transform() because of invisible characters 
-    so, before using Transform() remove invisible characters by using remove_invisible_chars()
-    for example self.play(Transform(remove_invisible_chars(text.chars[0:4]), remove_invisible_chars(text2.chars[0:2])))
+    """Display Text.
+
+    Text objects behave like a :class:`.VGroup`-like iterable of all characters
+    in the given text. In particular, slicing is possible.
```

Comment:
Suggestion : (if-minor) I think we should specify that this does not use LaTex.

Reply:
Yes, good catch! 1aab8955